gulpfile.js

```diff
@@ -37,5 +38,10 @@ gulp.task('tslint', () =>
         .pipe(tslint.report())
 );
 
-gulp.task('build', [ 'typescript', 'copydecs' ]);
+gulp.task('typedoc', () => {
+    exec('`npm bin`/typedoc --name "Resin ProcBots" --module commonjs --target ES6 --includeDeclarations ' +
```

Comment:
I'm not sure we do want to include declarations, both generally (except in a few specific cases, mainly just if you have a type definition for something you expect downstream users to implement to use your module) or in this module specifically (we don't need the github type defs in our docs).